### DISCUSSION TITLE
Vulkan binaries for Freya

### DISCUSSION THREAD

**Discussion by marc2332** [rust-skia/rust-skia#822] [General]
Hey Armin! :wave: 

Do you mind adding a **new** features-combo with Vulkan? I would like to test it with Freya, maybe I can get some performance improvements :zap: 

- Windows: `svg` + `textlayout` + `gl` + `vulkan` 
- macOS: `svg` + `textlayout` + `gl` + `vulkan`
- Linux:  `svg` + `textlayout` + `gl` + `x11` + `wayland` + `vulkan` 

Maybe I don't really need the `gl` feature if I am gonna be using `vulkan` ?

Let me know what you think, thanks again :D :pray: 

Marc

**Comment by pragmatrix** [comment]
Hello Marc, I'd suggest to add `vulkan` to the mix and remove the `gl` only ones. The binaries will only increase marginally in size, and `vulkan` does not add additional library dependencies, so even if you don't use `vulkan` in Freya, it should not make a noticeable difference.

> I would like to test it with Freya

For testing only, you should attempt to build rust-skia with the exact feature set you desire to test with first.

**Reply by marc2332** [reply]
> For testing only, you should attempt to build rust-skia with the exact feature set you desire to test with first.

Yeah, that's a fair point, I'll try to do it again, let's hope I don't fail 😅
